UI/NpcList/NPCRow.gd

```diff
@@ -0,0 +1,54 @@
+extends PanelContainer
+
+onready var _npcNameLabel = $HBoxContainer/Name
+onready var _npcGenderLabel = $HBoxContainer/Gender
+onready var _npcPersonalityLabel = $HBoxContainer/Personality
+onready var _amountOfChildrenLabel = $HBoxContainer/Children
+onready var meetButton = $HBoxContainer/CenterContainer2/Meet
+onready var forgetButton = $HBoxContainer/CenterContainer/Forget
+onready var showNpc = $ShowNPC
+var _npcID: String
+
+
+func setName(name: String):
+	_npcNameLabel.text = name
+
+
+func setGender(gender: String):
+	_npcGenderLabel.text = gender
+
+
+func setPersonality(personality: String):
+	_npcPersonalityLabel.text = personality
+
+
+func setChildren(children):
+	_amountOfChildrenLabel.text = String(children)
+
+
+func setNpcID(ID: String):
+	if(_npcID == null):
+		push_error("Exception: Attempt to set null NPC in the NPC list")
+	else:
+		_npcID = ID
+
+
+func getNpcID():
+	if(_npcID != null):
+		return _npcID
+	else: 
+		push_error("Exception: NPCRow: NPC ID is missing")
```

Comment:
use Log.error() instead, it will show up in the game's console then

Reply:
push_error( ) "Pushes an error message to Godot's built-in debugger and to the **OS terminal.**" That's probably better?

Reply:
Log.error is my function, it allows me to change the logging at any time without going through the whole codebase. Better to use the same function everywhere ^^